inc/theme-hooks.php

```diff
@@ -123,6 +123,53 @@ function alcatraz_output_sub_page_nav() {
 	}
 }
 
+add_action( 'alcatraz_entry_header_inside', 'alcatraz_output_default_entry_header' );
+/**
+ * Output the default entry header inner content.
+ *
+ * @since  1.0.0
+ *
+ * @param  int  $post_id  The current post ID.
+ */
+function alcatraz_output_default_entry_header( $post_id = 0 ) {
+
+	if ( ! $post_id ) {
+		$post_id = get_the_ID();
+	}
+
+	alcatraz_the_entry_title( $post_id );
+	alcatraz_the_entry_meta( $post_id );
+}
+
+add_action( 'alcatraz_entry_footer_inside', 'alcatraz_output_default_entry_footer' );
+/**
+ * Output the default entry footer inner content.
+ *
+ * @since  1.0.0
+ *
+ * @param  int  $post_id  The current post ID.
+ */
+function alcatraz_output_default_entry_footer( $post_id = 0 ) {
+
+	if ( ! $post_id ) {
+		$post_id = get_the_ID();
+	}
+
+	alcatraz_the_edit_post_link( $post_id );
+
+	echo '<hr>';
```

Comment:
Let's nix this...it borders on too opinionated, and I don't want to rewrite the footer just to remove it.